examples/lighting-app/efr32/BUILD.gn

```diff
@@ -46,7 +46,7 @@ declare_args() {
 show_qr_code = true
 
 # BRD4166A --> ThunderBoard Sense 2 (No LCD)
-if (efr32_board == "BRD4166A") {
+if (efr32_board == "BRD4166A" || efr32_board == "BRD4180A") {
```

Comment:
With the LCD module disabled for QR code , If using iOS/Android chip-tool , will the QR code string be displayed on the logs ? ( I believe it still does ) . Just wanted to double check

Reply:
Yes this is printed when LCD is disable
```
00> <info  > [SVR] SetupQRCode: [CH:Y3.1345600C0NF0ET00]
00> <info  > [SVR] Copy/paste the below URL in a browser to see the QR Code:
00> <info  > [SVR] https://dhrishi.github.io/connectedhomeip/qrcode.html?data=CH%3AY3.1345600C0NF0ET00
00> <info  > [SVR] Manual pairing code: [35249644644]
```